zarf-runner/rusty-rogers/zarf.yaml

```diff
@@ -0,0 +1,40 @@
+kind: ZarfPackageConfig
```

Comment:
Two thoughts on the Zarf runner
1. I believe we're planning to settle on a path forward for task runner, suspect this will be changed
2. Whatever the new path is will apply to more repos than UDS-Core, can we externalize it and reference it from here?

Maybe move the Zarf runner pieces to another PR so it is isolated from the Istio Capability?

Reply:
the zarf runner stuff is not generic right now. There are multiple considerations for how an individual capability might need to be deployed and tested

Reply:
Hi Mike. I agree its temporary and its super frustrating. As for your comments per [minimumcd](https://minimumcd.org/translations/texan/#continuous-integration)

```
The bare minimums for CI are:

    [Trunk-based development](https://minimumcd.org/translations/texan/#trunk-based-development)
    Work integrates into the trunk at least daily
    Y’all automated the testin’ before mergin'
    Work is tested with other work automagically on merge
    All feature work stops when the build halts and catches fire
    New work does not mess up delivered work
```
    
 I provided this translated version for Jeff. 
 
 Anyways, we shouldnt be merging to main without working ci.

Reply:
We state all over the place we follow this so 🤷